tests/surrealdb/README.md

```diff
@@ -0,0 +1,19 @@
+# SurrealDB test
+
+At the time of writing, we [cannot define schema at runtime for SurrealDB](https://github.com/surrealdb/surrealdb/issues/3541). We also [cannot define schema in a file and import it with single command](https://github.com/surrealdb/surrealdb/issues/3548).
```

Comment:
Does this solution work?
https://github.com/surrealdb/surrealdb/issues/3548#issuecomment-1962847770

Reply:
Just tested and it is working, I will update the doc now.